bundles/CoreBundle/src/EventListener/Frontend/ElementListener.php

```diff
@@ -143,20 +143,10 @@ private function handleAdminUserDocumentParams(Request $request, ?Document $docu
         }
 
         // document preview
-        if ($request->query->getBoolean('pimcore_studio_preview') || $request->query->getBoolean('pimcore_preview')) {
-            // get document from session
-
-            // TODO originally, this was the following call. What was in this->getParam('document') and
-            // why was it an object?
-            // $docKey = "document_" . $this->getParam("document")->getId();
-
-            if ($documentFromSession = Document\Service::getElementFromSession('document', $document->getId(), $request->getSession()->getId())) {
-                // if there is a document in the session use it
-                $this->logger->debug('Loading preview document {document} from session', [
-                    'document' => $document->getFullPath(),
-                ]);
-                $document = $documentFromSession;
-            }
+        if ($request->query->getBoolean('pimcore_studio_preview')) {
```

Comment:
Do we need to add this (`pimcore_studio_preview`) here: https://github.com/pimcore/skeleton/blob/2025.x/.docker/nginx.conf#L9 ?

Reply:
After sunsetting/removing the classic bundle I would rename the parameter again to simply pimcore_preview and just remove the session based stuff...